bin/local-env/install-wordpress.sh

```diff
@@ -64,31 +64,35 @@ fi
 
 # Create additional users.
 echo -e $(status_message "Creating additional users...")
-if [ ! "$(wp user get editor --field=login)" ]; then
-	wp user create editor editor@example.com --role=editor --user_pass=password --quiet
+
+if [[ $(wp user get editor --field=login 2>&1) != "editor" ]]; then
+	wp user create editor editor@example.com --role=editor --user_pass=password
 	echo -e $(status_message "Editor created! Username: editor Password: password")
 else
  echo -e $(status_message "Editor already exists, skipping...")
 fi
-if [ ! "$(wp user get author --field=login)" ]; then
+
+if [[ $(wp user get author --field=login 2>&1) != "author" ]]; then
 	wp user create author author@example.com --role=author --user_pass=password --quiet
 	echo -e $(status_message "Author created! Username: author Password: password")
 else
  echo -e $(status_message "Author already exists, skipping...")
 fi
-if [ ! "$(wp user get contributor --field=login)" ]; then
+if [[ $(wp user get contributor --field=login 2>&1) != "contributor" ]]; then
 	wp user create contributor contributor@example.com --role=contributor --user_pass=password --quiet
 	echo -e $(status_message "Contributor created! Username: contributor Password: password")
 else
  echo -e $(status_message "Contributor already exists, skipping...")
 fi
-if [ ! "$(wp user get subscriber --field=login)" ]; then
+if [[ $(wp user get subscriber --field=login 2>&1) != "subscriber" ]]; then
 	wp user create subscriber subscriber@example.com --role=subscriber --user_pass=password --quiet
 	echo -e $(status_message "Subscriber created! Username: subscriber Password: password")
 else
  echo -e $(status_message "Subscriber already exists, skipping...")
 fi
 
+wp user list
```

Comment:
@swissspidy I think we should leave this here. It is a good way to see in logs if user creation. Maybe we could format it look nicer in the logs..

I am also thinking about adding wp media list after attachments are made, so we can easily debug.